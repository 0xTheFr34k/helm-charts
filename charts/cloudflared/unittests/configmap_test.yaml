suite: test configmap

templates:
  - configmap.yaml

tests:
  - it: should match snapshot of default values
    set:
      tunnelConfig.name: unittest
    release:
      name: cloudflared
      namespace: cloudflared
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }

  - it: should match snapshot of warp routing on
    set:
      tunnelConfig.name: unittest
      tunnelConfig.warpRouting: true
    release:
      name: cloudflared
      namespace: cloudflared
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }
