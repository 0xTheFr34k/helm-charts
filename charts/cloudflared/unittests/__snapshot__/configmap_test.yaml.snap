should match snapshot of default values:
  1: |
    apiVersion: v1
    data:
      config.yaml: "\ntunnel: unittest\ncredentials-file: /etc/cloudflared/creds/credentials.json\noriginRequest:\n  connectTimeout: 30s\n\ningress: \n  - hostname: example.com\n    service: http://traefik.kube-system.svc.cluster.local:80\n  - service: http_status:404\n\nmetrics: 0.0.0.0:2000\nmetrics-update-freq: 5s\n\nautoupdate-freq: 24h\nno-autoupdate: true\n\ngrace-period: 30s\n\nretries: 5\n\n# auto, http2, h2mux, quic\nprotocol: auto\n\n# info, warn, error, fatal, panic\nloglevel: info\ntransport-loglevel: warn\n"
    kind: ConfigMap
    metadata:
      name: cloudflared
should match snapshot of warp routing on:
  1: |
    apiVersion: v1
    data:
      config.yaml: "\ntunnel: unittest\ncredentials-file: /etc/cloudflared/creds/credentials.json\noriginRequest:\n  connectTimeout: 30s\nwarp-routing:\n  enabled: true\n\ningress: \n  - hostname: example.com\n    service: http://traefik.kube-system.svc.cluster.local:80\n  - service: http_status:404\n\nmetrics: 0.0.0.0:2000\nmetrics-update-freq: 5s\n\nautoupdate-freq: 24h\nno-autoupdate: true\n\ngrace-period: 30s\n\nretries: 5\n\n# auto, http2, h2mux, quic\nprotocol: auto\n\n# info, warn, error, fatal, panic\nloglevel: info\ntransport-loglevel: warn\n"
    kind: ConfigMap
    metadata:
      name: cloudflared
